Tasks/HelmDeployV0/src/helmcommands/helmupgrade.ts

```diff
@@ -79,4 +81,11 @@ export function addArguments(helmCli: helmcli) : void {
         var chartPath = tl.getInput("chartPath", true);
         helmCli.addArgument("\"" + helmutil.resolvePath(chartPath)+ "\"");
     }
+
+    if(version) {
+        if(semver.valid(version))
+            helmCli.addArgument("--version ".concat(version));
+        else
+            tl.debug("The given version is not valid. Running the helm upgrade command with latest version");
+    }
```

Comment:
this seems like duplicate code.. Does it make sense to add this to the `helmCli` module?

Reply:
I think it would be inconsistent to put it in `helmCli` module because all the `addArgument` are happening here. But we can keep it in utils.